File: examples/TanksMiniGameProject/minigame/assets/src/System/IO/BinaryReader.js
Bridge.assembly("minigame-adaptor-project", function ($asm, globals) {
    "use strict";

    Bridge.define("System.IO.BinaryReader", {
        ctors: {
            ctor: function (input) {
                this.$initialize();
                throw new System.Exception("not impl");
            }
        },
        methods: {
            Read: function () {
                throw new System.Exception("not impl");
            },
            Read$1: function (buffer, index, count) {
                throw new System.Exception("not impl");
            },
            Read$2: function (buffer, index, count) {
                throw new System.Exception("not impl");
            },
            ReadBoolean: function () {
                throw new System.Exception("not impl");
            },
            ReadByte: function () {
                throw new System.Exception("not impl");
            },
            ReadBytes: function (count) {
                throw new System.Exception("not impl");
            },
            ReadChar: function () {
                throw new System.Exception("not impl");
            },
            ReadChars: function (count) {
                throw new System.Exception("not impl");
            },
            ReadDecimal: function () {
                throw new System.Exception("not impl");
            },
            ReadDouble: function () {
                throw new System.Exception("not impl");
            },
            ReadInt16: function () {
                throw new System.Exception("not impl");
            },
            ReadInt32: function () {
                throw new System.Exception("not impl");
            },
            ReadInt64: function () {
                throw new System.Exception("not impl");
            },
            ReadSByte: function () {
                throw new System.Exception("not impl");
            },
            ReadSingle: function () {
                throw new System.Exception("not impl");
            },
            ReadString: function () {
                throw new System.Exception("not impl");
            },
            ReadUInt16: function () {
                throw new System.Exception("not impl");
            },
            ReadUInt32: function () {
                throw new System.Exception("not impl");
            },
            ReadUInt64: function () {
                throw new System.Exception("not impl");
            }
        }
    });
});

File: unity-plugin/script-export/sdk/System~/IO/BinaryReader.cs
namespace System.IO {
    public class BinaryReader {
         public BinaryReader(Stream input) { throw new Exception("not impl"); }

        public virtual int Read() { throw new Exception("not impl"); }
        public virtual int Read(byte[] buffer, int index, int count) { throw new Exception("not impl"); }
        public virtual int Read(char[] buffer, int index, int count) { throw new Exception("not impl"); }

        public virtual bool ReadBoolean() { throw new Exception("not impl"); }
        public virtual byte ReadByte() { throw new Exception("not impl"); }
        public virtual byte[] ReadBytes(int count) { throw new Exception("not impl"); }
        public virtual char ReadChar() { throw new Exception("not impl"); }
        public virtual char[] ReadChars(int count) { throw new Exception("not impl"); }
        public virtual decimal ReadDecimal() { throw new Exception("not impl"); }
        public virtual double ReadDouble() { throw new Exception("not impl"); }
        public virtual short ReadInt16() { throw new Exception("not impl"); }
        public virtual int ReadInt32() { throw new Exception("not impl"); }
        public virtual long ReadInt64() { throw new Exception("not impl"); }
        public virtual sbyte ReadSByte() { throw new Exception("not impl"); }
        public virtual float ReadSingle() { throw new Exception("not impl"); }
        public virtual string ReadString() { throw new Exception("not impl"); }
        public virtual ushort ReadUInt16() { throw new Exception("not impl"); }
        public virtual uint ReadUInt32() { throw new Exception("not impl"); }
        public virtual ulong ReadUInt64() { throw new Exception("not impl"); }
    }
}